src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -76,7 +79,7 @@ public static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog, s
         /// <seealso cref = "ImageEstimatorsCatalog" />
```

Comment:
>/// <seealso cref = "ImageEstimatorsCatalog" /> [](start = 8, length = 47)

please remove

Reply:
it's on top of internal method, but will do.

---
In reply to: [276906321](https://github.com/dotnet/machinelearning/pull/3376#discussion_r276906321) [](ancestors = 276906321)